build: write a general overview explaining cryptocurrencies (#727)
Codecademy/docs: content/general/concepts/cryptocurrency/cryptocurrency.md

---
Title: 'Cryptocurrency'

> Description: 'A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralised networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world.'

Reviewer: ```suggestion
Description: 'A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Blockchain'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

> A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralised networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world. Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.

Reviewer: ```suggestion
A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world. Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.
```

Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.

### Types

> Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalisation. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.

Reviewer: ```suggestion
Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalization. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.
```

> Ethereum, the second most valuable cryptocurrency, features a smart contract platform that allows for numerous protocols including borrowing and lending services. There are also “stable coin” cryptocurrencies, which attempt to offer price stability and are backed by a reserve asset such as US dollars. There are also “altcoins”, some of which are forks of Bitcoin and others are new currencies, tokens or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the Doge meme created from a viral photo of a Shiba Inu.

Reviewer: ```suggestion
Ethereum, the second most valuable cryptocurrency, features a smart contract platform that allows for numerous protocols including borrowing and lending services. There are also “stable coin” cryptocurrencies, which attempt to offer price stability and are backed by a reserve asset such as US dollars. There are also “altcoins”, some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the Doge meme created from a viral photo of a Shiba Inu.
```

- _Transparency_: All cryptocurrency transactions are visible on the publicly distributed blockchain ledger and there are tools that allow anyone to look up the data for any transactions that have occurred on the network. It is also possible to audit the money supply to ensure that additional coins have not been illegitimately generated.
- _Security_: The only way to sign transactions or access the funds in your wallet is by gaining access to the private key of your crypto wallet. Furthermore, transactions are secured by the cryptographic nature of the blockchain and cannot be reversed like credit cards. There are, however, drawbacks to the nature of crypto assets discussed below.

### Disadvantages

- _Pseudo-anonymity_: While cryptocurrencies are not generally connected to an individual’s name, transactions on the blockchain can be tracked and traced. Furthermore, exchanges around the world almost universally require know-your-customer (KYC) regulation, which forces these institutions to collect identifying personal details before a customer is able to trade on their platform.

> - _Criminal activity_: Cryptocurrencies have the potential to be used by criminals, although the same is also true for physical cash. In the early days of Bitcoin, there was an online marketplace on the dark web called Silk Road which allowed users to purchase illegal drugs and conduct other nefarious activity.

Reviewer: ```suggestion
- _Criminal activity_: Cryptocurrencies have the potential to be used by criminals, although the same is also true for physical cash. In the early days of Bitcoin, there was an online marketplace on the dark web called Silk Road which allowed users to purchase illegal drugs and conduct other nefarious activities.
```

- _Decentralization_: While in theory, cryptocurrencies are meant to be decentralized, their ownership is often highly concentrated and the most value is generally accrued by the early investors. The statistic of highly skewed ownership is easily verifiable using blockchain data, and in one MIT study, it was demonstrated that just 11,000 investors held close to half of Bitcoin’s value.
- _Security_: Cryptocurrency blockchains are very secure through cryptography, though the wallets and exchanges where cryptocurrency assets are held can be hacked. One of the most famous exchange hacks occurred in 2014 when hackers accessed and stole over 700,000 bitcoins.
- _Price volatility_: Cryptocurrencies frequently experience high volatility, often changing value over the course of minutes or hours by more than those experienced in traditional financial markets over months or years. For example, on December 22, 2017, the price of Bitcoin dropped more than 30% in a single day from over US$16,000 to nearly US$11,000.

> - _Energy usage_: The mining of popular cryptocurrencies requires substantial energy costs, in some cases as much as entire countries. While that may be true, cryptocurrency supporters argue that existing financial structure uses far more energy than the power required for mining.

Reviewer: ```suggestion
- _Energy usage_: The mining of popular cryptocurrencies requires substantial energy costs, in some cases as much as entire countries. While that may be true, cryptocurrency supporters argue that the existing financial structure uses far more energy than the power required for mining.
```

---
Title: 'Cryptocurrency'
Description: 'Cryptocurrencies are digital assets or tokens secured by cryptography and transferred on decentralized networks based on blockchain technology.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Blockchain'
  - 'Cryptocurrency'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology.

Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.

### Blockchain

> Blockchain is essentially a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. Transactions are recorded in blocks that are linked together to form a chain, and these transactions are verified by nodes in the network. A node is a computer on the network running the software client that stores the blockchain data locally. The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.

Reviewer 2: ```suggestion
A blockchain is a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data. 

The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.
```

> Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.

Reviewer 2: ```suggestion
### Mining

Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.
```

#### Security

The only way to sign transactions or access the funds in your wallet is by gaining access to the private key of your crypto wallet. Furthermore, transactions are secured by the cryptographic nature of the blockchain and cannot be reversed like credit cards. There are, however, drawbacks to the nature of crypto assets discussed below.

### Disadvantages

#### Pseudo-anonymity

While cryptocurrencies are not generally connected to an individual’s name, transactions on the blockchain can be tracked and traced. Furthermore, exchanges around the world almost universally require know-your-customer (KYC) regulation, which forces these institutions to collect identifying personal details before a customer is able to trade on their platform.

#### Criminal activity

Cryptocurrencies have the potential to be used by criminals, although the same is also true for physical cash. In the early days of Bitcoin, there was an online marketplace on the dark web called Silk Road which allowed users to purchase illegal drugs and conduct other nefarious activities.

#### Decentralization

> While in theory, cryptocurrencies are meant to be decentralized, their ownership is often highly concentrated and the most value is generally accrued by the early investors. The statistic of highly skewed ownership is easily verifiable using blockchain data, and in one MIT study, it was demonstrated that just 11,000 investors held close to half of Bitcoin’s value.

Author: Great point, I will update it to refer to the Makarov and  Schoar (2021) paper you mentioned.

#### Security

> Cryptocurrency blockchains are very secure through cryptography, though the wallets and exchanges where cryptocurrency assets are held can be hacked. One of the most famous exchange hacks occurred in 2014 when hackers accessed and stole over 700,000 bitcoins.

Author: Interesting point. I have found that in the early days, when people held hundreds of coins, they often referred to them as countable "bitcoins". Nowadays, with it having become much more valuable and more like 'money', there seems to be a preference to use the singular 'Bitcoin'. I will update it to the more contemporary usage.

---
Title: 'Cryptocurrency'
Description: 'Cryptocurrencies are digital assets or tokens secured by cryptography and transferred on decentralized networks based on blockchain technology.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Blockchain'
  - 'Cryptocurrency'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology.

Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.

### Blockchain

> A blockchain is essentially a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data.

Reviewer 3: ```suggestion
A blockchain is essentially a set of interconnected "blocks" that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data.
```

The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.

### Mining

> Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.

Reviewer 3: ```suggestion
Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining. Nowadays, the computing power necessary is only achieved through a powerful network of computers known as a mining pool.
```

### Types

Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalization. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.

Ethereum, the second most valuable cryptocurrency, features a smart contract platform that allows for numerous protocols including borrowing and lending services. There are also “stable coin” cryptocurrencies, which attempt to offer price stability and are backed by a reserve asset such as US dollars.

> There are also “altcoins”, some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the Doge meme created from a viral photo of a Shiba Inu.

Reviewer 3: ```suggestion
There are also **altcoins**,some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the "Doge meme" created from a viral photo of a Shiba Inu.
```

I think we include inline meme names using double quotes

Reviewer 3: @KTom101 @Dusch4593 do we know if it's okay to bold words we want to bring attention to 💭

Author: Great, thanks for the feedback @yangc95. I'll wait until this is clarified then do all the changes at once.

Reviewer 2: @yangc95 @0xTomas Per our [Markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#bold), bold should be used for new terms, keywords, or concepts (but not for general emphasis).

Author: Thanks @Dusch4593, I have updated these changes.

> ### Purchasing cryptocurrencies

Reviewer 3: ```suggestion
### Purchasing Cryptocurrencies
```

The primary method to convert fiat currency into a cryptocurrency like Bitcoin is using an exchange. Cryptocurrency exchanges enable users to purchase crypto assets by funding their account using government-issued currencies such as the US dollar or the Euro and then trading on their platform. The exchange generates an online wallet where the assets are stored, although these assets can generally be withdrawn to a private wallet. Exchanges are almost universally required to comply with various Know Your Customer (KYC) regulations, which is an identity verification process that must be completed before a user purchases or sells cryptocurrencies.

> ## Advantages and disadvantages

Reviewer 3: ```suggestion
## Advantages and Disadvantages
```

> Cryptocurrencies were originally designed with the intent of revolutionizing existing financial infrastructure through greater efficiency and transparency combined with lower costs. There exists however a notable divide between the theoretical ideal of a transparent and decentralized system and its practical implementation in the real world.

Reviewer 3: ```suggestion
Cryptocurrencies were originally designed with the intent of revolutionizing existing financial infrastructure through greater efficiency and transparency combined with lower costs. There exists, however, a notable divide between the theoretical ideal of a transparent and decentralized system and its practical implementation in the real world.
```

> Outlined below are the primary advantages and disadvantages of cryptocurrencies.

Reviewer 3: ```suggestion
Outlined below are the primary advantages and disadvantages of cryptocurrencies:
```

#### Criminal activity

Like with physical cash, cryptocurrencies have the potential to be used by criminals. In the early days of Bitcoin, there was an online marketplace on the dark web called Silk Road which allowed users to purchase illegal drugs and conduct other nefarious activities.

#### Decentralization

While in theory, cryptocurrencies are meant to be decentralized, their ownership is often highly concentrated and the most value is generally accrued by the early investors. The statistic of highly skewed ownership is easily verifiable using blockchain data. In a study conducted by Makarov and Schoar (2021), conducted by the National Bureau of Economic Research, it was found around 10,000 individuals held about one-third of Bitcoin at that time.

#### Security

Cryptocurrency blockchains are very secure through cryptography, though the wallets and exchanges where cryptocurrency assets are held can be hacked. One of the most famous exchange hacks occurred in 2014 when hackers accessed and stole over 700,000 Bitcoin.

> #### Price volatility

Reviewer 3: ```suggestion
#### Price Volatility
```

The frequency and volatility of cryptocurrency price changes are remarkable when compared with traditional financial markets. For example, on December 22, 2017, the price of Bitcoin dropped more than 30% in a single day from over $16,000 USD to nearly $11,000 USD.

> #### Energy usage

Reviewer 3: ```suggestion
#### Energy Usage
```
